plugins/sagemaker/builtin/notebook.config

```diff
@@ -0,0 +1,8 @@
+[sdk]
```

Comment:
Why is this file necessary?

Reply:
what do you mean, are you saying why can we not use a common file somewhere else? 
This is the way I am connecting to Flyte from the notebook. A standardized config to access flyte environment and set some default values etc